app/views/auth/sessions/two_factor.html.haml

```diff
@@ -1,7 +1,7 @@
 - content_for :page_title do
   = t('auth.login')
 
-= javascript_pack_tag 'two_factor_authentication', crossorigin: 'anonymous'
+= vite_javascript_tag 'two_factor_authentication', crossorigin: 'anonymous'
```

Comment:
Should this have a file extension?

```suggestion
= vite_javascript_tag 'two_factor_authentication.js', crossorigin: 'anonymous'
```

Reply:
It's optional, but passing the extension would work (and is necessary if the script does not have a `js` extension, like `.jsx`).